pyproject.toml

```diff
@@ -148,6 +148,8 @@ filterwarnings = [
     "ignore:'H' is deprecated and will be removed in a future version. Please use 'h' instead of 'H':FutureWarning",
     'ignore:DataFrameGroupBy\.apply operated on the grouping columns\. This behavior is deprecated, and in a future version of pandas the grouping columns will be excluded from the operation\. Either pass `include_groups=False` to exclude the groupings or explicitly select the grouping columns after groupby to silence this warning\.:FutureWarning',
     "ignore:'BA' is deprecated and will be removed in a future version. Please use 'BY' instead of 'BA':FutureWarning",
+    'ignore:numpy\.core.* is deprecated and has been renamed to numpy\._core\.*:DeprecationWarning', # needed for pandas to import
+    'ignore:Passing a BlockManager to DataFrame is deprecated and will raise in a future version. Use public APIs instead:DeprecationWarning',
```

Comment:
This is something we have to address over in partd, can you open an issue there?

Reply:
issue opened here https://github.com/dask/partd/issues/77